workorder-recheck.lua

```diff
@@ -3,18 +3,66 @@
 
 workorder-recheck
 =================
-Sets the status to ``Checking`` (from ``Active``) of the selected work order (in the ``j-m`` or ``u-m`` screens).
+Sets the status to ``Checking`` (from ``Active``) of the selected work order.
 This makes the manager reevaluate its conditions.
 ]====]
-local scr = dfhack.gui.getCurViewscreen()
-if df.viewscreen_jobmanagementst:is_instance(scr) then
-    local orders = df.global.world.manager_orders
-    local idx = scr.sel_idx
-    if idx < #orders then
-        orders[idx].status.active = false
+
+--@ module = true
+
+local gui = require('gui')
+local widgets = require('gui.widgets')
+local overlay = require('plugins.overlay')
+
+local function set_current_inactive()
+    local scrConditions = df.global.game.main_interface.info.work_orders.conditions
+    if scrConditions.open then
+        local order = scrConditions.wq
+        order.status.active = false
     else
-        qerror("Invalid order selected")
+        qerror("Order conditions is not open")
     end
-else
-    qerror('Must be called on the manager screen (j-m or u-m)')
 end
+
+-- -------------------
+-- RecheckOverlay
+--
+
+local focusString = 'dwarfmode/Info/WORK_ORDERS/Conditions'
+
+RecheckOverlay = defclass(RecheckOverlay, overlay.OverlayWidget)
+RecheckOverlay.ATTRS{
+    default_pos={x=6,y=2},
+    default_enabled=true,
+    viewscreens=focusString,
+    frame={w=17, h=3},
+    frame_style=gui.MEDIUM_FRAME,
+    frame_background=gui.CLEAR_PEN,
+}
+
+function RecheckOverlay:init()
+    self:addviews{
+        widgets.HotkeyLabel{
+            frame={t=0, l=0},
+            label='recheck',
```

Comment:
Though I understand the need for making this label short, "recheck" might be too short for players to grok. I'd want some input from people on the Discord server about what would be best here.

Reply:
how about "re-check conditions" for this label? or maybe "request manager review"?

Reply:
"request conditions re-check"?

Reply:
or just "request re-check"?

Reply:
![image](https://github.com/DFHack/scripts/assets/348150/cec07efc-17a5-4f27-9166-7aab63f406d3)

I definitely want to keep the `check` there because it's everywhere on that screen